Algorithm.CSharp/AddRemoveOptionUniverseRegressionAlgorithm.cs

```diff
@@ -229,7 +229,7 @@ public override void OnSecuritiesChanged(SecurityChanges changes)
             {"Information Ratio", "0"},
```

Comment:
Do you know why the regressions are all 0? Should we edit the test?

Reply:
Good question, believe it's because of https://github.com/QuantConnect/Lean/blob/master/Common/Statistics/StatisticsBuilder.cs#L109, performance has a single sample that's getting removed, note that this regression algorithm goes for a single day